src/elements/Pagination/PaginationLink.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import style from './PaginationLink.scss'
```

Comment:
scss file name needs to be `./style.scss` since we will have one scss file per a component

Reply:
This is the nested component. `style.scss` has already existed. I considered to merge them together. But it's hard to obey the BEM rule after merge as layers are deep.
![image](https://user-images.githubusercontent.com/3041704/30569803-d6125faa-9c91-11e7-9b21-633043e35bbf.png)

Reply:
okay, got it.